ibu-imager/seedcreator/seedcreator.go

```diff
@@ -222,6 +226,76 @@ func (s *SeedCreator) stopServices() error {
 	return nil
 }
 
+func (s *SeedCreator) runRecertValidation() error {
+	s.log.Println("Running recert --force-expire tool and save a summary without sensitive data.")
+
+	// Get etcdImage available for the current release, this is needed by recert to
+	// run an unauthenticated etcd server for running successfully.
+	etcdImage, err := s.ops.GetImageFromPodDefinition("etcd")
+	if err != nil {
+		return err
+	}
+
+	// Run unauthenticated etcd server for the recert tool.
+	// This runs a small (fake) unauthenticated etcd server backed by the actual etcd database,
+	// which is required before running the recert tool.
+	s.log.Info("Run unauthenticated etcd server for recert dry-run")
+	_, err = s.ops.RunInHostNamespace(
+		"podman", []string{"run", "--name", "recert_etcd",
+			"--detach", "--rm", "--network=host", "--privileged", "--replace",
+			"--authfile", s.authFile, "--entrypoint", "etcd",
+			"-v", "/var/lib/etcd:/store",
+			etcdImage,
+			"--name", "editor",
+			"--data-dir", "/store"}...)
+	if err != nil {
+		return fmt.Errorf("failed to run recert_etcd container: %w", err)
+	}
+
+	// Ensure that the unauthenticated etcd server is killed before the runRecertValidation
+	// function returns, even if an error occurs.
```

Comment:
Comment feels a bit unnecessary,  cleanup regardless of error is what `defer` was created for, doesn't need to be explained